qiskit/providers/aer/backends/qasm_simulator.py

```diff
@@ -290,7 +290,7 @@ class QasmSimulator(AerBackend):
         'save_amplitudes', 'save_amplitudes_sq', 'save_state',
         'save_density_matrix', 'save_statevector', 'save_statevector_dict',
         'save_stabilizer', 'set_statevector', 'set_density_matrix',
-        'set_stabilizer'
+        'set_stabilizer', 'set_matrix_product_state'
```

Comment:
This shouldn't be added to the default instructions, since it is only supported on qasm simulator if you set the mps method.

The instruction also needs to be added to the mps method instructions in `aer_simulator.py`

Reply:
Done